src/app/zap-templates/zcl/data-model/chip/smoke-co-alarm-cluster.xml

```diff
@@ -0,0 +1,141 @@
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP" />
+
+  <cluster>
+    <name>Smoke CO Alarm</name>
+    <domain>Closures</domain>
+    <code>0x005C</code>
+    <define>SMOKE_CO_ALARM_CLUSTER</define>
+    <description>This cluster provides an interface for observing and managing the state of smoke and CO alarms.</description>
+
+    <!-- Abbreviations used in descriptions -->
+    <tag name="SMOKE" description="Smoke Alarm" />
+    <tag name="CO" description="CO Alarm" />
+
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+
+    <!-- Current cluster version -->
+    <globalAttribute side="either" code="0xFFFD" value="1" />
+
+    <!-- Attributes -->
+    <attribute side="server" code="0x0000" define="EXPRESSED_STATE" type="ExpressedStateEnum" writable="false" optional="false">ExpressedState</attribute>
+    <attribute side="server" code="0x0001" define="SMOKE_STATE" type="AlarmStateEnum" writable="false" optional="true">SmokeState</attribute>
+    <attribute side="server" code="0x0002" define="CO_STATE" type="AlarmStateEnum" writable="false" optional="true">COState</attribute>
+    <attribute side="server" code="0x0003" define="BATTERY_ALERT" type="AlarmStateEnum" writable="false" optional="false">BatteryAlert</attribute>
+    <attribute side="server" code="0x0004" define="DEVICE_MUTED" type="MuteStateEnum" writable="false" optional="true">DeviceMuted</attribute>
+    <attribute side="server" code="0x0005" define="TEST_IN_PROGRESS" type="boolean" writable="false" optional="false">TestInProgress</attribute>
+    <attribute side="server" code="0x0006" define="HARDWARE_FAULTALERT" type="boolean" writable="false" optional="false">HardwareFaultAlert</attribute>
+    <attribute side="server" code="0x0007" define="END_OF_SERVICEALERT" type="EndOfServiceEnum" writable="false" optional="false">EndOfServiceAlert</attribute>
+    <attribute side="server" code="0x0008" define="INTERCONNECT_SMOKE_ALARM" type="AlarmStateEnum" writable="false" optional="true">InterconnectSmokeAlarm</attribute>
+    <attribute side="server" code="0x0009" define="INTERCONNECT_CO_ALARM" type="AlarmStateEnum" writable="false" optional="true">InterconnectCOAlarm</attribute>
+    <attribute side="server" code="0x000A" define="CONTAMINATION_STATE" type="ContaminationStateEnum" writable="false" optional="true">ContaminationState</attribute>
+    <attribute side="server" code="0x000B" define="SENSITIVITY_LEVEL" type="SensitivityEnum" writable="true" optional="true">SensitivityLevel</attribute>
```

Comment:
This should list the min/max values, since it's writable.
```suggestion
    <attribute side="server" code="0x000B" define="SENSITIVITY_LEVEL" type="SensitivityEnum" writable="true" optional="true" min="0" max="2">SensitivityLevel</attribute>
```